From 0e105a546e0b920efc88c5f6cf1ee20cd6e35ea4 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:51:35 -0400
Subject: [fs] nfsd: Add missing comment of "expiry" in expkey_parse()

Message-id: <1403560314-32525-154-git-send-email-bfields@redhat.com>
Patchwork-id: 84574
O-Subject: [Patch RHEL7 153/172] NFSD: Add missing comment of "expiry" in expkey_parse()
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: f0db79d54b6c9f612fb2ef4f71ca8340edaf89f1
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/export.c b/fs/nfsd/export.c
index 858c536..263d30e 100644
--- a/fs/nfsd/export.c
+++ b/fs/nfsd/export.c
@@ -68,7 +68,7 @@ static struct svc_expkey *svc_expkey_lookup(struct cache_detail *cd, struct svc_
 
 static int expkey_parse(struct cache_detail *cd, char *mesg, int mlen)
 {
- /* client fsidtype fsid [path] */
+ /* client fsidtype fsid expiry [path] */
  char *buf;
  int len;
  struct auth_domain *dom = NULL;
-- 
1.7.1